### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

**Comment by rxu** [comment]
Looks fine.

File: event/listener.php [hunk 1]
```diff
@@ -251,13 +251,10 @@ public function viewonline_ideas($event)
 				$event['location_url'] = $this->helper->route('phpbb_ideas_index_controller');
 			}
 		}
-		else if ($event['on_page'][1] === 'viewtopic')
+		else if ($event['on_page'][1] === 'viewtopic' && $event['row']['session_forum_id'] == $this->config['ideas_forum_id'])
```
Comment (michaelcullum) [inline event/listener.php]: spacing?

Comment (VSEphpbb) [inline event/listener.php]: spacing??

Comment (michaelcullum) [inline event/listener.php]: `elseif` vs `else if`

Comment (VSEphpbb) [inline event/listener.php]: we use "else if"

Comment (michaelcullum) [inline event/listener.php]: Do we? Oh right. I knew that.

**Pull request merged**